src/praisonai/praisonai/gateway/server.py

```
@@ -1168,8 +1178,74 @@ async def reconnect_channel_handler(request) -> JSONResponse:
+            path = request.path_params.get("path", "")
+            hook = self.get_hook(path)
+            if hook is None or not getattr(hook, "enabled", True):
+                return JSONResponse({"error": "hook not found"}, status_code=404)
+
+            auth_err = _check_hook_auth(request, hook)
+            if auth_err:
+                return auth_err
+
+            try:
+                payload = await request.json()
+            except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)